x-pack/solutions/observability/plugins/slo/server/services/tasks/temp_summary_cleanup_task.ts

```diff
@@ -87,7 +87,7 @@ export class TempSummaryCleanupTask {
         params: {},
       });
     } catch (e) {
-      this.logger.error(`Error scheduling task, error: ${e}`);
+      this.logger.debug(`Error scheduling task, error: ${e}`);
```

Comment:
Just curious, what is the implication of such an error? Does the user need to know about this error, or do we handle it in a meaningful way?

Reply:
I was not sure about this one actually, maybe since it's a task, it is better to keep it as an error.